src/Field/Email.php

```diff
@@ -84,14 +88,31 @@ public function normalize($value)
         return parent::normalize(implode(', ', $emails));
     }
 
+    /**
+     * Validate DNS.
+     */
+    protected function isDNSValid(string $domain): bool
```

Comment:
What about one method like `hasAnyDnsRecord(string $domain, array $types = ['MX', 'A'])`?

Reply:
Can do like that but what it will change? :)

Reply:
A little cleanup, nothing more, no core functionality. I opened this mainly as this test was failing and occupying phpunit output as skipped :D